app/src/main/kotlin/app/vitune/android/ui/screens/home/HomeDiscovery.kt

```diff
@@ -106,6 +112,9 @@ fun HomeDiscovery(
         .padding(top = 24.dp, bottom = 8.dp)
         .padding(endPaddingValues)
 
+    val isDecember = Calendar.getInstance().get(Calendar.MONTH) == 4
```

Comment:
I'd recommend using kotlinx-datetime (already a dependency, so you can use it right away) for this, since `Calendar` is not consistent between Android SDK's.

Reply:
Fixed!